crates/bitcoin-da/src/helpers/builders.rs

```diff
@@ -519,7 +507,8 @@ pub fn create_inscription_type_0(
         // push nonce
         reveal_script_builder = reveal_script_builder
             .push_slice(nonce.to_le_bytes())
-            .push_opcode(OP_DROP);
+            // drop the second item, bc there is a big change it's 0 (tx kind) and nonce is >= 16
```

Comment:
```suggestion
            // drop the second item, bc there is a big chance it's 0 (tx kind) and nonce is >= 16
```
I think this is what was meant. Please change in all other places as well.

Reply:
good catch